components/camera/videosource/align.go

```diff
@@ -139,10 +139,10 @@ func getAligner(attrs *alignAttrs, logger golog.Logger) (transform.Aligner, erro
 // alignAttrs is the attribute struct for aligning.
 type alignAttrs struct {
 	*camera.AttrConfig
-	Color              string      `json:"color"`
-	Depth              string      `json:"depth"`
-	Height             int         `json:"height"`
-	Width              int         `json:"width"`
+	Color              string      `json:"color_url"`
+	Depth              string      `json:"depth_url"`
```

Comment:
These are not URLs, but are camera names already registered in the robot.
so they should be changed to something like "color_camera_name" and "depth_camera_name".